docs/user_guide/backup/index.md

```diff
@@ -43,7 +44,7 @@ tar --selinux -czvf config_files.tar.gz \
 /etc/sysconfig/elasticsearch \
 /root/ssl-build \
 /var/www/html/pub \
-/usr/share/katello/candlepin-cert.crt
+/var/lib/puppet/foreman_cache_data
 
```

Comment:
I believe some of the data files are outdated, and others are missing. When creating the backup scripts, these are the files I came up with:
tar --selinux -czvf config_files.tar.gz \
/etc/katello/ \
/etc/katello-installer/ \
/etc/candlepin/ \
/etc/foreman/ \
/etc/foreman-proxy/ \
/etc/gutterball/ \
/etc/hammer/ \
/etc/pulp/ \
/etc/puppet/ \
/etc/qpid/ \
/etc/qpid-dispatch/ \
/etc/elasticsearch/ \
/etc/tomcat6/ \
/etc/pki/katello/ \
/etc/pki/katello-certs-tools/ \
/etc/sysconfig/katello \
/etc/sysconfig/elasticsearch \
/etc/sysconfig/tomcat6 \
/root/ssl-build/ \
/var/lib/puppet/foreman_cache_data \
/var/www/html/pub/